scripts/operations/node_management/ncn-image-modification.sh

```diff
@@ -532,6 +532,37 @@ EOF
   done
 }
 
+function csm_145_patch {
+  local rpms=(
+    "${CSM_PATH}/rpm/cray/csm/sle-15sp4/noarch/dracut-metal-mdsquash-2.3.2-1.noarch.rpm"
```

Comment:
This is actually broken with what's in release/1.4 right now, the sibling `csm_144_patch` function has fix in a PR already that we should apply to this new `csm_145_patch` function in your PR.

https://github.com/Cray-HPE/docs-csm/pull/4977/files#diff-eb11136bd3f86c8829faa5f011fcc00abd7b28b56fe7b4669a469e76c1a3d716

This way `csm_145_patch` can merge fixed, asynchronously of that other PR.